astropy/stats/info_theory.py

```diff
@@ -379,7 +379,7 @@ def akaike_info_criterion_lsq(ssr, n_params, n_samples):
     >>> ssr_g2 = np.sum((g2_fit(x) - y)**2.0)
     >>> ssr_g1 = np.sum((g1_fit(x) - y)**2.0)
     >>> akaike_info_criterion_lsq(ssr_g3, 9, x.shape[0]) # doctest: +FLOAT_CMP
-    -656.32589850659224
+    -660.41075962620482
     >>> akaike_info_criterion_lsq(ssr_g2, 6, x.shape[0]) # doctest: +FLOAT_CMP
     -662.83834510232043
     >>> akaike_info_criterion_lsq(ssr_g1, 3, x.shape[0]) # doctest: +FLOAT_CMP
```

Comment:
@mirca , for some reason, `g3_fit` had negative `mean_1` without this PR and positive with this PR. Do you have any objection to the new results?

Without this PR:
```python
In [2]: g3_fit
Out[2]: <CompoundModel1(amplitude_0=0.1755099144937883, mean_0=0.2418419988277658, stddev_0=0.28246312142011093, amplitude_1=0.14903703265951587, mean_1=-3.057112188121801, stddev_1=0.30735643205583174, amplitude_2=2.451435082571759, mean_2=0.503285183744267, stddev_2=0.09909043268097108)>

In [3]: g2_fit
Out[3]: <CompoundModel2(amplitude_0=0.1753819504769808, mean_0=0.24045925337616975, stddev_0=0.28126103361662386, amplitude_1=2.4523765774919726, mean_1=0.5032999106348427, stddev_1=0.09912276546074991)>

In [4]: g1_fit
Out[4]: <Gaussian1D(amplitude=2.5329026083160606, mean=0.49935793058448075, stddev=0.10559425938501048)>
```

With this PR:
```python
In [2]: g3_fit
Out[2]: <CompoundModel1(amplitude_0=0.5965686483533806, mean_0=0.5093102217176415, stddev_0=0.5549516598914004, amplitude_1=-0.48405948352535877, mean_1=1.0170405002235874, stddev_1=0.9359042247635705, amplitude_2=2.3912809994262276, mean_2=0.5021418528282614, stddev_2=0.09698667234256535)>

In [3]: g2_fit
Out[3]: <CompoundModel2(amplitude_0=0.1753819504769808, mean_0=0.24045925337616975, stddev_0=0.28126103361662386, amplitude_1=2.4523765774919726, mean_1=0.5032999106348427, stddev_1=0.09912276546074991)>

In [4]: g1_fit
Out[4]: <Gaussian1D(amplitude=2.5329026083160606, mean=0.49935793058448075, stddev=0.10559425938501048)>
```